tests/functional/CoreTest.php

```diff
@@ -10,7 +10,7 @@
  *
  * @group commands
  */
-class CoreTest extends CommandUnishTestCase
+class FunctionalCoreTest extends CommandUnishTestCase
```

Comment:
is the difference in file name and class name required?

Reply:
I thought I renamed the file, must have missed it. Thanks for pointing that out.